Index: src/liblzma/Makefile.am
===================================================================
--- src/liblzma/Makefile.am	(revision 1)
+++ src/liblzma/Makefile.am	(working copy)
@@ -24,7 +24,7 @@
 	-I$(top_srcdir)/src/liblzma/delta \
 	-I$(top_srcdir)/src/liblzma/simple \
 	-I$(top_srcdir)/src/common
-liblzma_la_LDFLAGS = -no-undefined -version-info 0:0:0
+liblzma_la_LDFLAGS = -no-undefined -version-info 1:0:0
 
 include $(srcdir)/common/Makefile.inc
 include $(srcdir)/check/Makefile.inc
